lib/https.js

```diff
@@ -31,9 +31,15 @@ const inherits = util.inherits;
 const debug = util.debuglog('https');
 const { urlToOptions, searchParamsSymbol } = require('internal/url');
 
-function Server(opts, requestListener) {
+function Server(opts = {}, requestListener) {
   if (!(this instanceof Server)) return new Server(opts, requestListener);
 
+  if (typeof opts === 'function') {
+    requestListener = opts;
+    opts = undefined;
```

Comment:
The mean problem is the default value, removing that like https://github.com/nodejs/node/pull/13599/files#r121620077 works for me :)